operator/jsonnet/config.libsonnet

```
@@ -83,6 +97,10 @@ local utils = (import 'github.com/grafana/jsonnet-libs/mixin-utils/utils.libsonn
         if !std.member(dropList, r.record)
       ],
 
+    local dropHeatMaps = function(p)
```

Adding comment to these might help maintainability in the future from my POV. Hopefully I somewhat nailed the doc comment
```suggestion
    // dropHeatMaps filter function that returns "true" if a panel is of type "heatmap". To be used with function "dropPanels"
    local dropHeatMaps = function(p)
```